lib/daru/core/merge.rb

```diff
@@ -37,71 +37,67 @@ def hashify df
         def arrayify df
           arr = df.to_a
           col_names = arr[0][0].keys
-          values = arr[0].map{|h| h.values}
+          values = arr[0].map(&:values)
 
-          return col_names, values
+          [col_names, values]
         end
 
         def arrayify_with_sort_keys(size, df_hash, on)
-
           # Converting to a hash and then to an array is more complex
           # than using df.to_a or df.map(:row).  However, it's
           # substantially faster this way.
 
-          idx_keys = on.map { |key| df_hash.keys.index(key) }
+          # idx_keys = on.map { |key| df_hash.keys.index(key) }
```

Comment:
It is never used. I should've deleted it, in fact, but sometimes just commented out things that are not used.